packages/ui-view/src/ContextView/index.js

```diff
@@ -151,23 +161,18 @@ class ContextView extends Component {
       minWidth,
       margin,
       padding,
-      placement,
       shadow,
       stacking,
       style, // eslint-disable-line react/prop-types
-      textAlign
+      textAlign,
+      styles
     } = this.props
 
-    const classes = {
-      [styles.contextViewRoot]: true,
-      [styles[`placement--${placement.replace(' ', '-')}`]]: true
-    }
-
     return (
       <View
         {...omitProps(this.props, ContextView.propTypes)}
         style={style}
-        className={classnames(classes)}
+        css={styles.root}
```

Comment:
According to our agreement, classes should be named after the component, like "styles.contextView" in this case.

Reply:
Fixed